Sources/WhoopDIKit/Container/Container.swift

```diff
@@ -6,6 +6,10 @@ public final class Container {
 
     private let serviceDict = ServiceDictionary<DependencyDefinition>()
 
+    // Legacy local inject
+    private let localDependencyGraph: ThreadSafeDependencyGraph
+    private var isLocalInjectActive: Bool = false
```

Comment:
can we just add a comment here explaining that this uses the local dependency graph locking mechanism?